docs/java/index.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Let's please move this directory & file into the /docs/developers directory, so that this file lives at /docs/developers/java/index.md & will be accessible at ethereum.org/developers/java.

The reason being, our community has many ideas for adding additional content to this site. It'd be beneficial to add a bit more hierarchal structure to keep these pages contextual as developer-specific resources. It also gives the site a bit more flexibility for more resources to be added in the future.

Reply:
could we also set up a re-direct from ethereum.org/java, to ethereum.org/developers/java?

- easier to communicate to people, and easier for them to remember (e.g. in context of a conference, when you're trying to get a java developer to check this out afterwards)
- gives the subpage higher perceived importance

Reply:
Mmm ya, good points. We _could_ just keep it top-level, see how it goes & reorganize if things get unwieldy. I'm open to that.

Reply:
OK, I've left it at the top level for now.